src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -185,9 +189,7 @@ public function open()
 	    @stream_set_blocking($this->_socket, 0);
 	    $this->_currentProcessingMessage = '';
 	    //register_tick_function(array($this, 'process'));
-	    if ($this->_logger->isDebugEnabled()) {
```

Comment:
I don't understand what you are trying to say here. And it looks like some part of your comment got lost. 

The logger context parameter in PSR 3 has nothing to do with the log level. It is used to provide some meta data to the logger. Or I misunderstand your use of the word context here. 

Could you paste some sample code in the comment to help me understand?